src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -20,45 +21,51 @@
     :description                     (i18n/label :t/enable-notifications-sub-title)
     :description-accessibility-label :notifications-sub-title}])
 
+(defn- finish-onboarding
+  []
+  (rf/dispatch [:init-root :shell-stack])
+  (rf/dispatch [:profile/show-testnet-mode-banner-if-enabled])
+  (rf/dispatch [:universal-links/process-stored-event]))
+
+(defn- enable-notifications-and-navigate
+  []
+  (shell.utils/change-selected-stack-id shell.constants/default-selected-stack true nil)
+  (rf/dispatch [:request-permissions
+                {:permissions [:post-notifications]
+                 :on-allowed  #(log/info "push notification permissions were allowed")
+                 :on-denied   #(log/error "user denied push notification permissions")}])
+  (rf/dispatch [:push-notifications/switch true])
+  (finish-onboarding))
+
+(defn- skip-notifications-and-navigate
+  []
+  (shell.utils/change-selected-stack-id shell.constants/default-selected-stack true nil)
```

Comment:
this can be part of finish-onboarding itself

Reply:
I didn't do it because of the order of the calls, I wanted to preserve it. Do you think if I move this, the order will be preserved?

Reply:
Yes I think if you move this call in the beginning of `finish-onboarding` function before `:init-root :shell-stack`, it should work the same

Reply:
Thanks I'll apply it :+1: